localisation/00_wc_sad.csv

```diff
@@ -770,3 +772,44 @@ nick_twilight_of_the_gods;the Twilight of the Gods;;die G
 recruit_old_god_guard;Raise Old God's Guard;;Die Garde des alten Gottes ausheben;;;;;;;;;;;x
 recruit_old_god_guard_desc;The Old Gods can create legions of servants from their aberrant flesh and release them upon any fools who dare to face them.;;Die alten G�tter k�nnen Legionen von Dienern aus ihrem abartigen Fleisch erschaffen und sie auf jeden Narren loslassen, der es wagt, sich ihnen zu stellen.;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+become_brewmaster;Become a Brewmaster;;;;;;;;;;;;;x
+become_brewmaster_desc;Learn the secrets of making powerful brews.;;;;;;;;;;;;;x
+halaani_whiskey_intoxication;Halaani Whiskey Intoxication;;;;;;;;;;;;;x
+halaani_whiskey_intoxication_desc;Consumers find themselves talking a lot.;;;;;;;;;;;;;x
+sulfuron_slammer_intoxication;Sulfuron Slammer Intoxication;;;;;;;;;;;;;x
+sulfuron_slammer_intoxication_desc;It's gonna knock your socks off, then set your feet on fire.;;;;;;;;;;;;;x
+metal_brew_intoxication;Metal Brew Intoxication;;;;;;;;;;;;;x
+metal_brew_intoxication_desc;Heat of a burning forge with strong, stern tones of the earth.;;;;;;;;;;;;;x
+tiger_stout_intoxication;Tiger Stout Intoxication;;;;;;;;;;;;;x
+tiger_stout_intoxication_desc;Gives consumer swiftness of the tiger. Also makes him prone to meowing.;;;;;;;;;;;;;x
+serpent_ale_intoxication;Serpent Ale Intoxication;;;;;;;;;;;;;x
+serpent_ale_intoxication_desc;It's admirers often blabble about ssssecret knowledge.;;;;;;;;;;;;;x
+make_brew;Make Brew;;;;;;;;;;;;;x
+make_brew_desc;Brew a powerful alcoholic beverage.;;;;;;;;;;;;;x
+drink_brew;Drink Brew;;;;;;;;;;;;;x
+drink_brew_desc;Time to open one of your kegs.;;;;;;;;;;;;;x
+
+EVTDESC_WCLIFES.1;Brewmasters are known for making alcoholic drinks and concoctions that empower anyone that drinks them. The most famous brewmasters hail from the continent of Pandaria but Dwarves, Vrykuls and Ogres are also known for their brewing skills.;;;;;;;;;;;;;x
+EVTOPTA_WCLIFES.1;Time to learn the riddle of the brew.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTOPTA_WCLIFES.1;Time to learn the art of the brew.;;;;;;;;;;;;;x
```

Reply:
the riddle is the reference to https://conan.fandom.com/wiki/The_Riddle_of_Steel

Reply:
Fair enough, didn't know the reference.

Reply:
maybe replacing learn with study would work better. what do you think?

Reply:
I think it sounds better and fits the reference better

Reply:
Would work fine, and yeah, I think it sounds better too.